app/models/user.rb

```diff
@@ -468,6 +468,20 @@ def seed_aspects
     aq
   end
 
+  def send_welcome_message
+    unless AppConfig.settings.welcome_message? && AppConfig.admins.account? then return end
+    sender_username = AppConfig.admins.account.get
+    sender = User.find_by_username(sender_username)
+    conversation = sender.build_conversation(
+      participant_ids: [sender.person.id, self.person.id],
+      subject:         AppConfig.settings.welcome_message_subject.get,
+      message:         {text: AppConfig.settings.welcome_message_text.get.gsub("[USERNAME]",
+                              self.username)})
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant `self` detected.